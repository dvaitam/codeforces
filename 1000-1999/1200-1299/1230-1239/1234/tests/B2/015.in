26 10
14 6 4 15 6 8 6 4 14 13 18 10 18 9 16 11 4 7 11 2 1 1 10 20 11 15
