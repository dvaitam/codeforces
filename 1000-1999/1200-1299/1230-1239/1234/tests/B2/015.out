10
15 20 1 2 7 11 16 9 10 18
